params/hooks_libevm.go

```diff
@@ -195,6 +195,11 @@ func (a accessibleState) GetChainConfig() precompileconfig.ChainConfig {
 	return GetExtra(a.env.ChainConfig())
 }
 
+func (a accessibleState) GetRules() precompileconfig.Rules {
+	extra := GetExtra(a.GetPrecompileEnv().ChainConfig())
```

Comment:
nit: can we rename this to `chainConfigExtra` as there are actual `extras` package which can be confusing